Formula/regina-rexx.rb

```diff
@@ -5,21 +5,15 @@ class ReginaRexx < Formula
   sha256 "08e9a9061bee0038cfb45446de20766ffdae50eea37f6642446ec4e73a2abc51"
   license "GPL-2.0-or-later"
 
-  bottle do
-    sha256 arm64_monterey: "c97635f3169b3c26ebe86c3efb2cc591c936052f4b06a71144488e5917b911fd"
-    sha256 arm64_big_sur:  "959d6f80f93d13720130111f1f898b97b521affcbbc2b67b8540d5e518512a86"
-    sha256 monterey:       "4bc6b21a904b70ae6775da17b948d54b0e1e78849d330c7777c1896b540cb0a9"
-    sha256 big_sur:        "1c37cd2d7e0860ebabb477de8a89fb9debedcfabb5ccffcabfba1eef17b18a62"
-    sha256 catalina:       "d132bc2537c7b8a07cd31fb9dc11c77e460243fd0bac5b6f69d3ab32e06f53cc"
-    sha256 x86_64_linux:   "4f461dda1b7fc8a862c87509ad52cf525f6b4c02c15d33e07f5f7274a20e808c"
-  end
-
-  uses_from_macos "libxcrypt"
-
   def install
     ENV.deparallelize # No core usage for you, otherwise race condition = missing files.
-    system "./configure", *std_configure_args
-    system "make", "install"
+    system "./configure", "--disable-debug",
+                          "--prefix=#{prefix}",
+                          "--with-addon-dir=#{HOMEBREW_PREFIX}/lib/regina-rexx/addons",
+                          "--with-brew-addon-dir=#{lib}/regina-rexx/addons"
+    system "make"
+    system "make", "installbase", "installbrewlib"
+    ln_s "libregina3.pc", lib/"pkgconfig/libregina3.pc"
```

Comment:
What are you trying to do here? Doesn't this create a `libregina3.pc` symlink pointing to itself?

Reply:
Yeah.  My bad.  Removed.